src/app.js

```diff
@@ -25,11 +38,72 @@ const quoteData = [
   },
 ];
 
+// const quoteList = new QuoteList();
+let quoteTemplate;
+
+
 $(document).ready(function() {
+  let bus = {};
+
+  bus = _.extend(bus, Backbone.Events);
+
   const quotes = new QuoteList(quoteData);
   const simulator = new Simulator({
     quotes: quotes,
   });
+  // const $quoteList = $('#quotes');
+  //
+  // const quoteView = new QuoteView({
+  //   model: quotes.at(0),
+  //   template: _.template($('#quote-template').html()),
+  //   tagName: 'li',
+  //   className: 'quote',
+  // });
+  // THESE ARE THE THINGS THAT BACKBONE EXPECTS except for template, which is why we initialized template.
+
+  // $quoteList.append(quoteView.render().$el);
+
 
   simulator.start();
+
+
+  quoteTemplate = _.template($('#quote-template').html());
+
+  const quoteListView = new QuoteListView({
+    //this is what params becomes
+    el: '#quotes',
+    model: quotes, //need explanation
+    template: quoteTemplate,
+    tradeTemplate:  _.template($('#trade-template').html()),
+    bus: bus,
+  });
+
+  const order = new Order;
+  const orders = new OrderList();
+
+  const orderListView = new OrderListView({
+    el: '#orders',
+    model: orders,
+    template: _.template($('#order-template').html()),
+  });
+
+
+
+  const orderFormView = new OrderFormView({
+    el: '#order-entry-form',
+    orderList: orders,
+    quoteList: quotes
```

Comment:
I like that the `OrderFormView` knows about the `orderList` and the `quoteList`. This makes sense - these are the things it needs in order to do its job. I saw many students attempt to use the bus to get this information around, which ends up being a little clunky.